modules/ocf_desktop/files/stats/update-flags.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+# state can be "active" "cleanup"
+# "active" implies a current logged-in session
+# "cleanup" triggers closing of the session
+# other files to look at: session-cleanup and session-setup in /etc/lightdm
+
+CUR_USER=$(who | awk '$NF == "(:0)" { print $1 }')
```

Comment:
Some impressive `awk`ing, but `whoami` is much simpler here

Reply:
the original script runs as the `ocfstats` user, so `whoami` wouldn't work. which is why we parse the user logged into the primary display for stats.